packages/replayio/src/commands/record.ts

```diff
@@ -47,6 +48,10 @@ async function record(url: string = "about:blank") {
 
         writeFileSync(errorLogPath, stderr, "utf8");
 
+        sendEvent("cli-recorder-crash", {
+          message: stderr,
+        });
```

Comment:
I think we'd probably want to send more info along with such a crash– the runtime build id for example, and potentially the CLI package version as well.

There may also be [crash dump files](https://chromium.googlesource.com/crashpad/crashpad) with data that isn't present in `stderr`. Chromium puts them [here](https://www.chromium.org/developers/crash-reports/#generating-crash-dumps-locally) by default but I'm not sure where Replay writes them. (This is something I left a question about on PRO-235/TT-983.)